irispy/new_sji.py

```diff
@@ -0,0 +1,209 @@
+'''
+This module provides movie tools for level 2 IRIS SJI fits file
+'''
+
+from datetime import timedelta
+
+from ndcube import NDCube
+import numpy as np
+from astropy.io import fits
+import astropy.units as u
+from astropy.wcs import WCS
+from sunpy.time import parse_time
+from sunpy import config
+from irispy import iris_tools
+
+__all__ = ['SJICube']
+
+TIME_FORMAT = config.get("general", "time_format")
+
+# the following value is only appropriate for byte scaled images
+BAD_PIXEL_VALUE = -200
```

Comment:
I think to be clear, this variable should be renamed ```BAD_PIXEL_VALUE_SCALED```.

Reply:
Modified